core/domain/user_jobs_one_off_test.py

```diff
@@ -1216,9 +1216,8 @@ def test_that_last_created_time_is_updated(self):
             self.exp_id, self.owner_id, end_state_name='End')
         self.logout()
 
-        user_settings = user_services.get_user_settings(self.owner_id)
-        user_settings.last_created_an_exploration = None
```

Comment:
Why was this line removed?

Reply:
@kevinlee12 This line used for setting `user_settings`'s `last_created_an_exploration` to `None` and is no longer needed as I passed `owner_id` to `record_user_created_an_exploration` in which it is set to None.